<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="31" total="33" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Automation" duration-ms="374728" started-at="2018-02-02T20:47:31Z" finished-at="2018-02-02T20:53:46Z">
    <groups>
    </groups>
    <test name="TestScripts" duration-ms="374728" started-at="2018-02-02T20:47:31Z" finished-at="2018-02-02T20:53:46Z">
      <class name="com.testscripts.TestCases">
        <test-method status="PASS" signature="openApp()[pri:0, instance:com.testscripts.TestCases@10a035a0]" name="openApp" is-config="true" duration-ms="10803" started-at="2018-02-02T15:47:20Z" finished-at="2018-02-02T15:47:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApp -->
        <test-method status="PASS" signature="addBeneficiaryPersonal()[pri:0, instance:com.testscripts.TestCases@10a035a0]" name="addBeneficiaryPersonal" duration-ms="258430" started-at="2018-02-02T15:47:31Z" finished-at="2018-02-02T15:51:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addBeneficiaryPersonal -->
        <test-method status="FAIL" signature="addBeneficiaryBusiness()[pri:0, instance:com.testscripts.TestCases@10a035a0]" name="addBeneficiaryBusiness" duration-ms="116266" started-at="2018-02-02T15:51:49Z" finished-at="2018-02-02T15:53:46Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Expected condition failed: waiting for presence of element located by: By.xpath: //*[@content-desc='TransferHeader'] (tried for 60 second(s) with 500 MILLISECONDS interval)]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for presence of element located by: By.xpath: //*[@content-desc='TransferHeader'] (tried for 60 second(s) with 500 MILLISECONDS interval)at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:82)at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:231)at com.library.Common.waitUntilElementPresent(Common.java:161)at com.businessfunctions.TransferPage.addBeneficiaryToAccount(TransferPage.java:213)at com.testscripts.TestCases.addBeneficiaryBusiness(TestCases.java:406)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate an element using By.xpath: //*[@content-desc='TransferHeader']For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.htmlBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'System info: host: 'MGGG0HH2', ip: '10.149.250.196', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_152'Driver info: driver.version: unknownat org.openqa.selenium.support.ui.ExpectedConditions.lambda$findElement$0(ExpectedConditions.java:882)at java.util.Optional.orElseThrow(Optional.java:290)at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:881)at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:43)at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:182)at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:179)at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)... 28 more]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addBeneficiaryBusiness -->
        <test-method status="PASS" signature="closeApp()[pri:0, instance:com.testscripts.TestCases@10a035a0]" name="closeApp" is-config="true" duration-ms="23825" started-at="2018-02-02T15:53:46Z" finished-at="2018-02-02T15:54:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- com.testscripts.TestCases -->
    </test> <!-- TestScripts -->
  </suite> <!-- Automation -->
</testng-results>
